servicebus/spring-cloud-azure-stream-binder-servicebus/servicebus-queue-binder-arm/README.md

```diff
@@ -0,0 +1,186 @@
+# Spring Cloud Azure Stream Binder for Service Bus queue by ARM Authentication Sample shared library for Java
+
+
+This code sample demonstrates how to use the Spring Cloud Stream binder for
+Azure Service Bus queue by Azure Resource Manager Authentication. 
+
+
+## What You Will Build
+You will build an application using Spring Cloud Stream to send and receive messages for Azure Service Bus Queue, and  a service principal for this application.
```

Comment:
You will build an application using Spring Cloud Stream to send and receive messages for Azure Service Bus Queue. This sample will
- Create a Service Bus queue and a service principal and assign the `Contributor` role of the service bus to the service principal.
- Connect to the ARM using the service principal and retrieve the connection string of the service bus.
- Connect to the Service Bus queue to send and receive messages.